kqueen/server.py

```diff
@@ -55,6 +56,8 @@ def create_app(config_file=None):
 
 
 app = create_app()
+# setup Bcrypt
+bcrypt = Bcrypt(app)
```

Comment:
This is probably causing circual depency mentioned above. This initialization of `Bcrypt` in server module looks insane to me.

What is advantage of using `flask-bcrypt` instead of bare `bcrypt`? 

https://pypi.python.org/pypi/bcrypt/3.1.0 - sectip password hashing. This approach don't need this silly "global" variables.